docs/source/installation.rst

```diff
@@ -83,127 +83,81 @@ Note: not all soft dependencies of ``sktime`` are also available on ``conda-forg
-
-.. code-block:: bash
-
-    pip install build
-    python -m build --wheel
-
-The resulting packages are generated in the ``dist/`` folder.
-
-Contributor or 3rd party extension developer setup
---------------------------------------------------
+The following steps guide you through the process:
 
 1. Follow the Git workflow: Fork and clone the repository as described in [Git and GitHub workflow](https://www.sktime.net/en/stable/developer_guide/git_workflow.html)
 
 2. Set up a new virtual environment. Our instructions will go through the commands to set up a ``conda`` environment which is recommended for sktime development.
-This relies on an `anaconda installation <https://www.anaconda.com/products/individual#windows>`_. The process will be similar for ``venv`` or other virtual environment managers.
+The process will be similar for ``venv`` or other virtual environment managers.
+
+  .. warning::
+       Using ``conda`` via one of the commercial distributions such as Anaconda
+       is in general not free for commercial use and may incur significant costs or liabilities.
+       Consider using free distributions and channels for package management,
+       and be aware of applicable terms and conditions.
 
-In the ``anaconda prompt`` terminal:
+In the ``conda`` terminal:
 
 3. Navigate to your local sktime folder, :code:`cd sktime` or similar
 
-4. Create a new environment with a supported python version: :code:`conda create -n sktime-dev python=3.8` (or :code:`python=3.11` etc)
+4. Create a new environment with a supported python version: :code:`conda create -n sktime-dev python=3.11` (or :code:`python=3.12` etc)
 
    .. warning::
-       If you already have an environment called "sktime-dev" from a previous attempt you will first need to remove this.
+       If you already have an environment called ``sktime-dev`` from a previous attempt you will first need to remove this.
 
 5. Activate the environment: :code:`conda activate sktime-dev`
 
 6. Build an editable version of sktime.
-In order to install only the dev dependencies, :code:`pip install -e .[dev]`
+In order to install only the dev dependencies, :code:`pip install -e ".[dev]"`
 If you also want to install soft dependencies, install them individually, after the above,
-or instead use: :code:`pip install -e .[all_extras,dev]` to install all of them.
-
-    .. note::
+or instead use: :code:`pip install -e ".[all_extras,dev]"` to install all of them.
 
-        If this step results in a "no matches found" error, it may be due to how your shell handles special characters.
-
-        - Possible solution: use quotation marks:
-
-            .. code-block:: bash
-
-                pip install -e ."[dev]"
-
-7. If everything has worked you should see message "successfully installed sktime"
+7. If everything has worked, you should see message "successfully installed sktime"
 
 Some users have experienced issues when installing NumPy, particularly version 1.19.4.
 
-
```

Comment:
Above comment about Numpy 1.19.4 should probably be removed. `sktime` currently needs `1.21` or above.

Reply:
yes, this document seems quite old.

I am wondering, is this duplicating something? I recall updating install instructions a while ago, but do not see my updates here.

Reply:
Not sure. You are probably considering the file where I recommended separate braches for separate works, as I remember you refactored it later to cover more points.

Reply:
no, I was thinking of sth different, but I only remember vaguely. Perhaps not useful to dwell on this further.